src/careamics/losses/loss_factory.py

```diff
@@ -35,92 +28,6 @@ class FCNLossParameters:
     loss_weight: float
 
 
-@dataclass  # TODO why not pydantic?
-class LVAELossParameters:
```

Comment:
Can you comment on where do all these parameters went and how they are now passed to the loss?

Reply:
It was deprecated, replaced but what we now have in `src/careamics/config/loss_model.py`.